dockerfiles/is/Dockerfile

```diff
@@ -23,8 +23,12 @@ ENV DEBIAN_FRONTEND noninteractive
 # set user information
 ARG USER=wso2user
 ARG USER_GROUP=wso2
+ARG USER_ID=1000000000
```

Comment:
It might be better to refer IS Dockerfile available here [1] and update accordingly. We have done a collection of improvements and was evaluating the possibility of using a single Docker image across all container platforms. 

[1] https://github.com/wso2/docker-is/blob/master/dockerfiles/is/Dockerfile

Reply:
@imesh +1.

@Megala21 it would be great if you can observe how this has been done in Kubernetes resources for WSO2 Enterprise Integrator [1] and Docker resources for WSO2 Enterprise Integrator [2].

[1]: [Kubernetes resources for WSO2 Enterprise Integrator](https://github.com/wso2/kubernetes-ei)
[2]: [Docker resources for WSO2 Enterprise Integrator](https://github.com/wso2/docker-ei/tree/master/dockerfiles)